DM_2/orders/orders_model.js

```diff
@@ -1,129 +1,114 @@
-      return orderCustomerInfo.rows[0];
-    } catch (e) {
-      console.log(e);
+      for (let prod of this.products) {
+        await OrderItems.create({
+          order_id: this.order_id,
+          product_id: prod.id,
+          quantity: prod.count,
+        });
+      }
+    } catch (err) {
+      throw new DefaultError(400, err.stack);
     }
   }
 
-  async getOrderInfo(user) {
+  async getOrderInfo() {
     try {
-      const queryOrderInfo = `
-      SELECT products.id, order_item.quantity 
-      FROM products, order_item 
-      WHERE order_item.order_id = $1 AND order_item.product_id = products.id`;
-      const orderInfo = await client.query(queryOrderInfo, [this.order_id]);
-      this.order = orderInfo.rows;
-      return {user: user, products: this.order};
-    } catch (e) {
-      console.log(e);
+      const order = await OrderItems.findAll({
+        attributes: [['product_id', 'id'], 'quantity'],
+        where: {order_id: this.order_id},
+      });
+      this.order = order.map(el => el.dataValues);
+      return {products: this.order};
+    } catch (err) {
+      throw new DefaultError(400, err.stack);
     }
   }
 
   async getDetailOrderInfo() {
     try {
-      const queryOrderDetailInfo = `
-      SELECT products.id, products.product_name, products.price, products.amount, units.unit, order_item.quantity, products.price * order_item.quantity sum 
-      FROM products, order_item, units 
-      WHERE order_item.order_id = $1 AND order_item.product_id = products.id AND units.id = products.id_units`;
-      const orderDetailInfo = await client.query(queryOrderDetailInfo, [this.order_id]);
-      const prodData = orderDetailInfo.rows;
+      let prodsArr = [];
       let noProdErr = [];
-      for(let prod of prodData) {
-        if (prod.amount < prod.quantity) {
-          const data = {id: prod.id, amount: prod.amount};
+      for (let item of this.order) {  // На это нагромождение не обращайте внимания, пока не особо разобралась как делать красивые запросы в sequelize. Исправлюсь))
+        const prodValues = await Products.findAll({where: {id: item.id}});
+        const product = prodValues[0].dataValues;
+        const unit = await Units.findAll({attributes: ['unit'], where: {id: product.id_units}});
+        const manufacture = await Manufactures.findAll({
+          attributes: ['manufacture'],
+          where: {id: product.id_manufacture},
+        });
+        const category = await Categories.findAll({attributes: ['category'], where: {id: product.id_category}});
+        const quantity = await OrderItems.findAll({
+          attributes: ['quantity'],
+          where: {product_id: product.id, quantity: item.quantity},
+        });
+        product.unit = unit[0].dataValues.unit;
+        product.category = category[0].dataValues.category;
+        product.manufacture = manufacture[0].dataValues.manufacture;
+        product.quantity = quantity[0].dataValues.quantity;
```

Comment:
You can use `.findOne()` method :)

[Sequelize Documentation](https://sequelize.org/v3/docs/models-usage/)

```
Project.findOne({
  where: {title: 'aProject'},
  attributes: ['id', ['name', 'title']]
})
```